test/integration/targets/aws_codebuild/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+# tasks file for aws_codebuild
+
+- name: Run aws_codebuild integration tests.
+
+  block:
+
+    # ==================== preparaions ========================================
+
+    - name: set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+      no_log: yes
+
+    - name: create IAM role needed for CodeBuild
```

Comment:
This role should probably be cleaned up after the tests are done. (Also, I don't have any experience specifically with CodeBuild, but based on general experience with AWS you may need to insert a short pause after the IAM role is created to avoid sporadic failures when trying to use a newly created role.)

Reply:
Done